monitoring/grafana.yaml

```diff
@@ -0,0 +1,91 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: grafana-pvc
+  namespace: monitoring
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: grafana
+  name: grafana
+  namespace: monitoring
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: grafana
+  template:
+    metadata:
+      labels:
+        app: grafana
+    spec:
+      securityContext:
+        fsGroup: 472
+        supplementalGroups:
+          - 0
+      containers:
+        - name: grafana
+          image: acraks6jqf6qyt7jarg.azurecr.io/grafana/grafana:7.3.0
```

Comment:
If you had to import the grafana image into your ASB acr, this will be helpful to add in the doc.

Reply:
We shouldn't be hard coding the image. It is best to use a placeholder and use `envsubst`

Reply:
Instructions updated to use envsubst and grafana.yaml is now a template that gets copied to gitops folder